sandpack-client/src/utils.ts

```diff
@@ -31,22 +34,59 @@ export function addPackageJSONIfNeeded(
 ): SandpackBundlerFiles {
   const newFiles = { ...files };
 
-  if (!newFiles["/package.json"]) {
-    if (!dependencies) {
-      throw new Error(
-        "No dependencies specified, please specify either a package.json or dependencies."
-      );
-    }
+  const getPackageJsonFile = (): string | undefined => {
+    if (newFiles["/package.json"]) return "/package.json";
+    if (newFiles["package.json"]) return "package.json";
```

Comment:
Should we normalise these paths at some point? Just a note for potential future improvements

Reply:
Yes, indeed we need to address it. I noted this problem when I was trying to implement a new feature, which fetches sandboxes from CodeSandbox, and I faced a ton of issues (see `resolveFile` for instance).

Here's the task: https://github.com/codesandbox/sandpack/issues/353